WordPress/gutenberg
# [RNMobile]  Set a min height on the inserter menu

## Description

Sets the `setMinHeightToMaxHeight` on the bottom sheet in the inserter menu.

Gutenberg Mobile PR https://github.com/wordpress-mobile/gutenberg-mobile/pull/3914

Fixes #34481 

## How has this been tested?




With the demo app on both iOS and Android:
1. Tap on the inserter '+' 
2. The block list should be scrollable ( on Android the inserter will be fullscreen, on iOS only partial height)
3. Activate the search by taping the search input
4. The list should remain scrollable.
3. Add a block that supports inner blocks i.e. Buttons
4. While inside the button block open the inserter
5. The inserter should open fullscreen and not be empty


## Screenshots 

Android

https://user-images.githubusercontent.com/744755/131913164-63971840-59a3-4bf0-9fab-2ad3beed0be3.mp4 

iOS

https://user-images.githubusercontent.com/744755/131913292-714b4726-0c4b-41dc-add9-047ab4b23d39.mp4





## Types of changes





## Checklist:
